chain/client/tests/cross_shard_tx.rs

```diff
@@ -448,16 +450,21 @@ mod tests {
 
     #[test]
     fn test_cross_shard_tx() {
-        test_cross_shard_tx_common(64, false);
+        test_cross_shard_tx_common(64, false, false);
     }
 
     #[test]
     fn test_cross_shard_tx_8_iterations() {
-        test_cross_shard_tx_common(8, false);
+        test_cross_shard_tx_common(8, false, false);
+    }
+
+    #[test]
+    fn test_cross_shard_tx_8_iterations_drop_chunks() {
+        test_cross_shard_tx_common(8, false, true);
```

Comment:
Add a test with 64 iterations as well and run it couple times to completion. 8 iterations often is not enough to expose bugs.